test/integration/connection-monitoring-and-pooling/connection.test.js

```diff
@@ -16,7 +16,7 @@ describe('Connection', function () {
 
   describe('Connection - functional/cmap', function () {
     it('should execute a command against a server', {
-      metadata: { requires: { apiVersion: false } },
+      metadata: { requires: { apiVersion: false, topology: '!load-balanced' } },
```

Comment:
https://github.com/mongodb/node-mongodb-native/blob/main/test/tools/runner/filters/mongodb_topology_filter.js

Unfortunately this "not" logic isn't implemented in the topology filter, so this is just filtering on all topologies since it does exact string matches. Maybe this is a good time to add it. The os_filter does support this so we could use the same logic from there.

Reply:
I've added to the topology filter now.